altiProfil/install/www/altiprofil/js/altiProfil.js

```diff
@@ -172,11 +172,12 @@ function getProfil(p1,p2){
           };
         var data = [profilLine];
 
-
+        var plotLocale = navigator.language || navigator.userLanguage;
```

Comment:
May not be the best way , ploty expect `<lang_region> ` values ("en_US") and navigator.language can return simply lang value ("fr"). You can rely on the lizmap locale value to set this variable

`$lizmapLocale = jApp::config()->locale` 
on php side and build the js variable using the [DockContent template](https://github.com/arno974/lizmap-altiProfil/blob/master/altiProfil/templates/altiProfil_Dock.tpl)